test/ctia/test_helpers/store.clj

```diff
@@ -11,13 +11,26 @@
                    helpers/fixture-ctia
                    es-helpers/fixture-delete-store-indexes])})
 
-(s/defn test-for-each-store-with-app
+(s/defn test-selected-stores-with-app
   "Takes a 1-argument function which accepts a Trapperkeeper `app`
-  which should succeed for all stores."
-  [t :- (s/=> s/Any
+  which should succeed for the stores named in the first argument."
+  [selected-stores
+   t :- (s/=> s/Any
               (s/named s/Any 'app))]
-  (doseq [[store-key fixtures] store-fixtures]
+  (assert (set? selected-stores))
```

Comment:
I would rather add a schema check for selected-stores and assert here that's it's not empty. WDYT?